src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -82,9 +85,218 @@ private static (ISymbolSlot slot, FormulaValue value) AddBlankVar(SymbolTable sy
 
         private static PowerFxConfig AllEnumsSetup(PowerFxConfig config)
         {
-            return PowerFxConfig.BuildWithEnumStore(new EnumStoreBuilder().WithDefaultEnums(), new TexlFunctionSet(), config.Features);
+            var store = new EnumStoreBuilder().WithDefaultEnums();
+            var newConfig = PowerFxConfig.BuildWithEnumStore(store, new TexlFunctionSet(), config.Features);
+
+            // There are likewise no built in functions that take Boolean backed option sets as parameters
+            newConfig.AddFunction(new TestXORBooleanFunction());
+            newConfig.AddFunction(config.Features.StronglyTypedBuiltinEnums ? new STE_TestXORYesNoFunction() : new Boolean_TestXORYesNoFunction());
+            newConfig.AddFunction(new TestColorInvertFunction());
+            newConfig.AddFunction(config.Features.StronglyTypedBuiltinEnums ? new STE_TestColorBlueRampInvertFunction() : new Color_TestColorBlueRampInvertFunction());
+

```

Comment:
Ideally those should be in a different setup. Power Apps can run tests with the default built-in enums. It wont be able to run the test with those new functions.

Reply:
I split the tests into two files, one that uses the old AllEnumsSetup and one that uses the new one with the test enums that covers Boolean and Color enums.  Should be good to go for Power Apps.